core/src/test/scala/unit/kafka/server/BrokerRegistrationRequestTest.scala

```diff
@@ -57,7 +57,7 @@ class BrokerRegistrationRequestTest {
 
         val saslMechanism: String = ""
 
-        def isZkController: Boolean = !clusterInstance.isKRaftTest
+        def isZkController: Boolean = false
```

Comment:
Could we remove this property directly ?

Reply:
To maintain code consistency here, I prefer to remove this until zk is removed from `ControllerInformation`.